### ISSUE TITLE
Markdown: unstable lists

### ISSUE THREAD

**Issue by SimenB** [prettier/prettier#3168]
**Prettier 1.8.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiVACAygVwA54QBO8AJhgEo4A2cAzgDpRMC0GA2lBC6QJZ0BDAEa1SLeHRh0AugAoA9HgFgA1gIDm9efWq9YLPNRzq9LAFb0Y80hDB15RGlq49+w0eMt0AdAFtSAJQYbHyC1NQQAO4Yoe5w5BJS3qwcLgBmtjh08Z6SMgpKqhpaOnowBkYmUOaW1rb2jrT26ZnZYok+-kEhbuFRGBlgWfEYHcnVqdy8pAgwvGAC1OK8MLRyispqmval+obGphaSdXYOTs1TM7Dzi8urcH6BwTG9EdHTszfUoytN42zsABui2mLDgAA88HAwDB1oUtiU6Lo9pVDrUbKdGlpgboxBCoTDHt0MAg6DgiHASZDoTAMPwMAtwiNIEQKTDqABPcZMdDYfCEEgjKhNJhMABUk1cghEOQ6cM2xR2SLKFQO1SOVgxDXO8hcsRl7S8RKYGGesT60X1olGRvFksGw0NeXlRW22mVKLVNWOWrOTV13AdbVySS6JrNr36QZGYztnEun0ZdzWBQVbt25X2VW9mvqfucCeuSbm92NUFNPTCbzpVzmjJ+9x8cZxoPxNJdCKVyMzqPV6LzWPsLbx1MJXWC4dJ5MpbZhdLoDMW1pZbNWXJAABoQBA8HNoHRkKABJJkGlFtkt0IiJs4DAsIU9OpkDBHHAtxjT+e3yAzHRwQAha9VFvLABF8OAABk9DgT9qAvbccBgPBEIAJlg+ClCIbIiGQEBfAEIgVBsSIoE3EA8CIMoAHVphgAALZAAA4AAYt2yXxeGfV82MfWgAEUcAgeB0O-GBhBo0h6OQFCtxfAReGRdQAGEIF8fDcK4KAYK3YYABVhAPJAzzguAAF9TKAA)
```sh
--parser markdown
```

**Input:**
```markdown
## Supported Rules

- [no-disabled-tests](/packages/eslint-plugin-jest/docs/rules/no-disabled-tests.md) - disallow disabled tests.
- [no-focused-tests](/packages/eslint-plugin-jest/docs/rules/no-focused-tests.md) - disallow focused tests.
- [no-identical-title](/packages/eslint-plugin-jest/docs/rules/no-identical-title.md) - disallow identical titles.
- [valid-expect](/packages/eslint-plugin-jest/docs/rules/valid-expect.md) - ensure expect is called correctly.

## Supported Rules

* [no-disabled-tests](/packages/eslint-plugin-jest/docs/rules/no-disabled-tests.md)
  - disallow disabled tests.
* [no-focused-tests](/packages/eslint-plugin-jest/docs/rules/no-focused-tests.md)
  - disallow focused tests.
* [no-identical-title](/packages/eslint-plugin-jest/docs/rules/no-identical-title.md)
  - disallow identical titles.
* [valid-expect](/packages/eslint-plugin-jest/docs/rules/valid-expect.md) -
  ensure expect is called correctly.
```

**Output:**
```markdown
## Supported Rules

* [no-disabled-tests](/packages/eslint-plugin-jest/docs/rules/no-disabled-tests.md)
  - disallow disabled tests.
* [no-focused-tests](/packages/eslint-plugin-jest/docs/rules/no-focused-tests.md)
  - disallow focused tests.
* [no-identical-title](/packages/eslint-plugin-jest/docs/rules/no-identical-title.md)
  - disallow identical titles.
* [valid-expect](/packages/eslint-plugin-jest/docs/rules/valid-expect.md) -
  ensure expect is called correctly.

## Supported Rules

* [no-disabled-tests](/packages/eslint-plugin-jest/docs/rules/no-disabled-tests.md)
  * disallow disabled tests.
* [no-focused-tests](/packages/eslint-plugin-jest/docs/rules/no-focused-tests.md)
  * disallow focused tests.
* [no-identical-title](/packages/eslint-plugin-jest/docs/rules/no-identical-title.md)
  * disallow identical titles.
* [valid-expect](/packages/eslint-plugin-jest/docs/rules/valid-expect.md) -
  ensure expect is called correctly.

```

**Expected behavior:**
No ~change~ breaking of input, as it creates sublists

**Comment by ikatyang** [comment]
Thanks for the report, didn't realize it'll break in those cases, `-/*/>/+` should be considered unbreakable.

**Comment by azz** [comment]
This has been fixed in `1.8.1`.
